Update ssh-logging.md (#10515)
cloudflare/cloudflare-docs: content/cloudflare-one/policies/gateway/network-policies/ssh-logging.md

2. In the **Network** tab, create a new network policy.

> 3. Name the policy and specify the [Destination IP](/cloudflare-one/policies/gateway/network-policies/#destination-ip) for your origin server. This can be done using a Public or Private IP. Please refer to [Configuring a Private Network](https://developers.cloudflare.com/cloudflare-one/connections/connect-networks/private-net/connect-private-networks/) to use a Private IP.

Reviewer: ```suggestion
3. Name the policy and specify the [Destination IP](/cloudflare-one/policies/gateway/network-policies/#destination-ip) for your origin server. 

You can enter either a public or private IP. To use a private IP, refer to [Connect private networks](/cloudflare-one/connections/connect-networks/private-net/connect-private-networks/).
```
